### PR TITLE
Update 03_configuration.adoc

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

File: documentation/modules/ROOT/pages/03_configuration.adoc [hunk 1]
```diff
@@ -1,7 +1,5 @@
 = Configuration
 
-:project-name: fruits-app
```
Comment (gsmet) [inline documentation/modules/ROOT/pages/03_configuration.adoc:3]: This seems unused and not in line with the naming of the tutorial anyway.

File: documentation/modules/ROOT/pages/03_configuration.adoc [hunk 2]
```diff
@@ -121,9 +119,9 @@ public class GreetingResourceTest {
 }
 ----
 
-NOTE: If you're still running in live testing mode you will already see that the tests pass again and you can ignore the rest of this page and move on to the next step.
+NOTE: If you're still running in continuous testing mode, you will already see that the tests pass again and you can ignore the rest of this page and move on to the next step.
 
-Stop your current Live Coding session of Quarkus in the terminal by sending a `CTRL+C`:
+Stop your current Live Coding session of Quarkus in the terminal by sending a kbd:[Ctrl+C]:
```
Comment (gsmet) [inline documentation/modules/ROOT/pages/03_configuration.adoc:124]: Again, we need to make sure this is supported by the tooling.

Comment (kdubois) [reply documentation/modules/ROOT/pages/03_configuration.adoc:124]: Looks like the kdb: doesn't render correctly: 
image
We'll have to change the 02_basics page back too.

Comment (gsmet) [reply documentation/modules/ROOT/pages/03_configuration.adoc:124]: OK, that's odd. It's supposed to be supported: https://docs.antora.org/antora/latest/asciidoc/ui-macros/#keybinding and is not new.

I tried to upgrade to latest Antora and same result. I will revert that change and fix this PR too.

Comment (gsmet) [reply documentation/modules/ROOT/pages/03_configuration.adoc]: @kdubois I reverted the kbd change here too.

**Comment by kdubois** [comment]
Thanks!

**Pull request merged**